build_projects/dotnet-cli-build/Chmod.cs

```diff
@@ -47,12 +33,12 @@ protected override string GenerateFullPathToTool()
 
         protected override string GenerateCommandLineCommands()
         {
-            return $"{GetRecursive()} {GetMode()} {GetFilePath()}";
+            return $"{GetRecursive()} {GetMode()} {GetGlob()}";
         }
 
-        private string GetFilePath()
+        private string GetGlob()
```

Comment:
Nitpick: Why the extra indirection, as opposed to calling `Glob` in `GenerateCommandLineCommands`?